Docker/Dockerfile

```diff
@@ -12,6 +12,8 @@ ENV container docker
 
 ADD * /opt/conda/bin/
 
+CMD mv /opt/conda/bin/treehandler.py /opt/conda/lib/python3.9/site-packages/eukcc/treehandler.py
```

Comment:
Where did treehandler.py come from?

Reply:
From the EukCC repo, https://github.com/EBI-Metagenomics/EukCC/blob/master/eukcc/treehandler.py and add quoted node name option to the Tree method.